src/Enum.php

```diff
@@ -188,6 +188,16 @@ public static function toArray(): array
         return array_combine(static::getValues(), static::getIndices());
     }
 
+    /**
+     * @return \Spatie\Enum\Enumerable[]
+     */
+    public static function getAll(): array
+    {
+        return array_map(function(string $value) {
+            return static::make($value);
+        }, static::getValues());
```

Comment:
```suggestion
        }, static::getIndices());
```

Reply:
That suggestion breaks the test.

Reply:
Feel free to change it to what you feel is best 👍

Reply:
https://github.com/spatie/enum/pull/33/commits/051fa365f34e2c25b1d28c9a5e85ca628815e86c